### PR TITLE
Update cortana-integration.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected from US only to worldwide as this feature is currently being rolled out (Roadmap - https://www.microsoft.com/en-us/microsoft-365/roadmap?filters=Rolling%20out&searchterms=57204)

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3636

**Comment by rsun0525** [comment]
When will this show up?  I'm still seeing the phrasing https://docs.microsoft.com/en-us/microsoft-365/admin/misc/cortana-integration?view=o365-worldwide "New Microsoft 365 experiences, such as the Briefing email and Play My Emails, will be enabled using Cortana enterprise services and fully comply with those promises. These features are currently available in the English (United States) market for users with mailboxes in the US geo, and for employees of multi-national corporations with US-based mailboxes. For more information on finding the usage location, please visit View additional property values for accounts."

**Comment by msbemba** [comment]
@rsun0525 This is a Pull request created for the author to review and approve (it takes 2-3 business days). Once the author Approves the Change it takes less than 48 hours for the changes to reflect on the document.

**Comment by rsun0525** [comment]
Thanks for the detail.  Have a great day.

File: microsoft-365/admin/misc/cortana-integration.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Moving forward, we're focusing Cortana on enterprise productivity.
...
 
-- New Microsoft 365 experiences, such as the Briefing email and Play My Emails, will be enabled using Cortana enterprise services and fully comply with those promises. These features are currently available in the English (United States) market for users with mailboxes in the US geo, and for employees of multi-national corporations with US-based mailboxes. For more information on finding the usage location, please visit [View additional property values for accounts](https://docs.microsoft.com/microsoft-365/enterprise/view-user-accounts-with-microsoft-365-powershell?view=o365-worldwide#view-additional-property-values-for-accounts).
+- New Microsoft 365 experiences, such as the Briefing email and Play My Emails, will be enabled using Cortana enterprise services and fully comply with those promises. These features are currently available Worldwide (Standard Multi-Tenant). For more information on finding the usage location, please visit [View additional property values for accounts](https://docs.microsoft.com/microsoft-365/enterprise/view-user-accounts-with-microsoft-365-powershell?view=o365-worldwide#view-additional-property-values-for-accounts).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/cortana-integration.md]:
```suggestion
- New Microsoft 365 experiences, such as the Briefing email and Play My Emails, will be enabled using Cortana enterprise services and fully comply with those promises. These features are currently available worldwide (standard multi-tenant). For more information on finding the usage location, please visit [View additional property values for accounts](https://docs.microsoft.com/microsoft-365/enterprise/view-user-accounts-with-microsoft-365-powershell?view=o365-worldwide#view-additional-property-values-for-accounts).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks.

**Pull request merged**